logging.md

```diff
@@ -0,0 +1,40 @@
+# Logging
+
+We want traceability from HTTP calls.
+- Each HTTP call needs to generate a call ID (preferable on the proxy). This ID must be should be pass to the scope of the ZIO application. So every log will mention this ID. This ID must also be returned to the user as a HTTP header.
+- In case of errors and problem reports those IDs can be use by the user to get support.
+
+Level 3 support can be provided to user during the logging retention policies. We user need to provide the value in the header `X-Request-Id` to get this support.
```

Comment:
A user can receive Level 3 support during the logging retention policies. The user needs to provide the value in the header `X-Request-Id` to get this support.